### DISCUSSION TITLE
An error occurred during replacement

### DISCUSSION THREAD

**Discussion by gengdz** [LazyVim/LazyVim#1223] [Q&A] [answered]
I was using `<leader>sr` for global replacement, and when I used `<leader>R`, the following error occurred

![image](https://github.com/LazyVim/LazyVim/assets/41912568/52719e04-96cd-43c2-a997-7dbe7e83f67d)

![image](https://github.com/LazyVim/LazyVim/assets/41912568/2faaaa27-80b1-42ea-a7c2-f81edc838c33)

asking for help～

**Comment by folke** [comment, marked as answer]
The error literally tells you that it can't find gsed. Just install it.

**Reply by gengdz** [reply]
it works, thank you so much!

I solved the problem by running the following command: `brew install gnu-sed`
